Final_Project/Ship.cpp

```diff
@@ -0,0 +1,151 @@
+#include "Ship.h"
+
+Ship::Ship(float x, float y) : 
+    moving(false), 
+    speed(100.f), 
+    health(100), 
+    damage(15), 
+    maxShield(0), 
+    shield(0),
+    maneuverability(15),
```

Comment:
At some point it would be best to move all const params like that into some kind of Config file for easier balance configs and content making